utils/docker/Dockerfile.jammy

```diff
@@ -40,6 +40,12 @@ RUN mkdir /ms-playwright && \
     npm i /tmp/playwright-core.tar.gz && \
     npm exec --no -- playwright-core mark-docker-image "${DOCKER_IMAGE_NAME_TEMPLATE}" && \
     npm exec --no -- playwright-core install --with-deps && rm -rf /var/lib/apt/lists/* && \
+    # Workaround for https://github.com/microsoft/playwright/issues/27313
+    if [ "$(uname -m)" = "aarch64" ]; then \
+        rm /usr/lib/aarch64-linux-gnu/gstreamer-1.0/libgstwebrtc.so; \
```

Comment:
this will only delete symlink, not libgstwebrtc.1.0.23.so, but I guess it's enough to fix our issue.